wiki/Game_modifier/Score_multiplier/en.md

```diff
@@ -0,0 +1,26 @@
+---
```

Comment:
Are you still working on this?

Reply:
How the hell did this go unnoticed? Apologies, I'll finish it as soon as I can.